pkg/utils/commands.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import "fmt"
+
+type Commad struct {
+	Name        string
+	Description string
+}
+
+var availableCommands = []Commad{
+	{"digger help", "Display help information"},
+	{"digger version", "Display version information"},
+	{"digger apply", "Apply digger configuration"},
+	{"digger plan", "Plan digger configuration"},
```

Comment:
it doesn't apply digger configuration, but terraform configuration. It just adds few more things on top, like locking